### PR TITLE
schemas added

### PR DESCRIPTION
**Pull request by yoonited1**

### CODE REVIEW

File: schemas/model_link.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      ZoomLink: {
+        type: String
+      },
+      ClassPage: {
+        type: String
+      },
+      PiazzaLink: {
+        type: String
+      },
+      OtherLink: {
```
Comment (DualSaturn) [inline schemas/model_link.js:16]: Is it possible to create an array of "other links"?

File: schemas/model_class.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const { Decimal128 } = require("bson");
+const { link } = require("fs");
+const mongoose = require("mongoose");
+
+let Class = new Schema(
+    {
+      ClassName: {
```
Comment (DualSaturn) [inline schemas/model_class.js:7]: Do we want each class to have an associated quarter/semester?

File: schemas/model_calendar.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        type: Date
+      },
+      Class: ClassSchema,
+      HaveClass: {
+        type: Boolean
+      },
+      ClassTime: {
+        type: String
+      },
+      Assignment: AssignmentSchema,
```
Comment (DualSaturn) [inline schemas/model_calendar.js:17]: Is this the schema for a single event in the calendar?

File: schemas/model_link.js (hunk 1 above)
Comment (yoonited1) [reply schemas/model_link.js:16]: Yes, it is possible. Please take a look at the change I made.

**Pull request merged**